Announce.txt

```diff
@@ -25,11 +25,18 @@ RELEASE VERSION/DATE TO BE FILLED IN LATER
   since last release.  This announcement highlights only the important
   changes.
 
-  NOTE: The 4.0.0 Release of SCons will drop Python 2.7 Support
-
   Please note the following important changes since release 3.1.1:
-      - Added debug option "action_timestamps" which outputs to stdout the absolute start and end time for each target.
-      - Update Command() function to accept target_scanner, source_factory, and target_factory arguments.
+      - Python 2.7 is not longer supported. SCons now requires Python 3.5 or later.
```

Comment:
Since 3.1.2 maybe?  there seem to be several minor releases missing anyway (not your fault) - @bdbaddog, what to do with those?

Reply:
This may be discarded given that the changes in RELEASES.txt turned out to be the important part.